library/src/main/resources/SAMLSpecRefMessage.properties

```diff
@@ -293,87 +288,78 @@ SAMLBinding.doc=Bindings for the OASIS Security Assertion Markup Language (SAML)
   Composite
 SAMLBinding.uri=https://www.oasis-open.org/committees/download.php/56780/sstc-saml-bindings-errata-2.0-wd-06-diff.pdf
 
-SAMLBindings_3_1_2_1=servers MUST authenticate to clients using a X.509 v3 certificate
+SAMLBindings_3_1_2_1_a=servers MUST authenticate to clients using a X.509 v3 certificate
 
 SAMLBindings_3_4_3_a=RelayState data MAY be included with a SAML protocol message transmitted with \
   this binding. The value MUST NOT exceed 80 bytes in length
 
-SAMLBindings_3_4_3_a1=The [RelayState] value MUST NOT exceed 80 bytes in length
+SAMLBindings_3_4_3_b=The [RelayState] value MUST NOT exceed 80 bytes in length
 
-SAMLBindings_3_4_3_b1=If a SAML request message is accompanied by RelayState data, then the SAML \
-  responder...MUST place the exact data it received with the request into the corresponding \
+SAMLBindings_3_4_3_c=If a SAML request message is accompanied by RelayState data, then the SAML \
+  responder MUST return its SAML protocol response using a binding that also supports a RelayState \
+  mechanism, and it MUST place the exact data it received with the request into the corresponding \
   RelayState parameter in the response.
 
 SAMLBindings_3_4_4_a=A URL encoding MUST place the message entirely within the URL query string, \
   and MUST reserve the rest of the URL for the endpoint of the message recipient
 
-SAMLBindings_3_4_4_1=Any signature on the SAML protocol message, including the <ds:Signature> XML \
+SAMLBindings_3_4_4_1_a=Any signature on the SAML protocol message, including the <ds:Signature> XML \
   element itself, MUST be removed
 
-SAMLBindings_3_4_4_1_a=The compressed data is subsequently base64-encoded according to the rules \
+SAMLBindings_3_4_4_1_b=The compressed data is subsequently base64-encoded according to the rules \
   specified in IETF RFC 2045 [RFC2045]. Linefeeds or other whitespace MUST be removed from the \
   result.
 
-SAMLBindings_3_4_4_1_a1=The compressed data is subsequently base64-encoded according to the rules \
-  specified in IETF RFC 2045 [RFC2045].
-
-SAMLBindings_3_4_4_1_a2=Linefeeds or other whitespace MUST be removed from the [base64 encoded] \
-  result.
-
-SAMLBindings_3_4_4_1_b1=The base-64 encoded data is then URL-encoded
+SAMLBindings_3_4_4_1_c=The compressed data is subsequently base64-encoded according to the rules \
```

Comment:
✏️ Should we merge this one with `SAMLBindings_3_4_4_1_b`? I don't think they're different enough to merit different error codes.